### ISSUE TITLE
Mario isn't visible, but Lara is

### ISSUE THREAD

**Issue by Ultracat3** [headshot2017/OpenLara#52]
When I got everything ok to start the game, it did it normally, no issues, but when I gained movement Mario wasn't visible, only it's shadow, I tried getting a second player on to see if it fixed, but it didn't, what do I do?

**Comment by headshot2017** [comment]
download [oglver](https://frank-buss.de/opengl/oglver32.zip) and show me a screenshot of your opengl version

**Comment by Ultracat3** [comment]
![image](https://user-images.githubusercontent.com/119438006/221005587-b1992d1d-6903-4623-8dd7-f8db2f6ef8d4.png)

**Comment by headshot2017** [comment]
try this version of the game with the console enabled and show me what the console spits out after mario spawns in: [https://cdn.discordapp.com/attachments/1013375382667403365/1013580819924779100/OpenLara_mingw.exe](https://cdn.discordapp.com/attachments/1013375382667403365/1013580819924779100/OpenLara_mingw.exe)
(place the exe in the openlara folder)

**Comment by Ultracat3** [comment]
![image](https://user-images.githubusercontent.com/119438006/221014131-3444c04b-7c86-41e9-84bf-bc7b08931bf4.png)

**Comment by headshot2017** [comment]
move mario around and show me another screenshot

**Comment by Ultracat3** [comment]
![image](https://user-images.githubusercontent.com/119438006/221014996-35fd9a07-386a-4e77-bbd3-f3642ae85785.png)
Like this?

**Comment by headshot2017** [comment]
yeah - from what i see, Mario appears to be fine in the exe i sent you but not in the release version?

**Comment by Ultracat3** [comment]
Do I just try again in the release version?

**Comment by Ultracat3** [comment]
![image](https://user-images.githubusercontent.com/119438006/221015801-e1332307-3bdc-4f48-8d69-8a5ba4335d7e.png)
See?

**Comment by headshot2017** [comment]
that is bizarre - i'll quickly compile an exe of version 1.02 with the console enabled and post it here, since the one i sent you is for 1.0

**Comment by Ultracat3** [comment]
Ok

**Comment by headshot2017** [comment]
[OpenLara_mingw.zip](https://github.com/headshot2017/OpenLara/files/10817888/OpenLara_mingw.zip)

**Comment by Ultracat3** [comment]
Do I replace the other exe?

**Comment by headshot2017** [comment]
yes

**Comment by Ultracat3** [comment]
It shows up an error

**Comment by headshot2017** [comment]
open cmd.exe on that folder, run OpenLara_mingw.exe and post a screenshot

**Comment by Ultracat3** [comment]
how do I open cmd.exe on that folder?
(I have cmd open)

**Comment by Ultracat3** [comment]
Do I tell you what the error says?
(It's In Spanish, is it ok?)

**Comment by headshot2017** [comment]
it's fine - spanish is my main language, but for the sake of others reading this let's try to keep our conversation in english

**Comment by headshot2017** [comment]
in cmd, type "cd", press space, drag the openlara folder to the cmd window and press enter, then type OpenLara_mingw.exe
![image](https://user-images.githubusercontent.com/36455681/221019276-15674232-fcaf-45fc-b904-82caaf7a0869.png)

**Comment by Ultracat3** [comment]
Ok
![image](https://user-images.githubusercontent.com/119438006/221019421-6f268a75-531a-4edc-b131-f2e7c37baf2a.png)
Do I translate it here?

**Comment by headshot2017** [comment]
oh - i'll need to send you an updated sm64.dll too. give me a minute

**Comment by Ultracat3** [comment]
ok

**Comment by Ultracat3** [comment]
I have to replace it with the one that it's already in the folder right?

**Comment by headshot2017** [comment]
yes
[sm64.zip](https://github.com/headshot2017/OpenLara/files/10818121/sm64.zip)

**Comment by Ultracat3** [comment]
![image](https://user-images.githubusercontent.com/119438006/221022627-d12e0bdf-e32e-4a1c-a3b6-ec45fa05c7b9.png)

**Comment by Ultracat3** [comment]
sorry got confused

**Comment by Ultracat3** [comment]
![image](https://user-images.githubusercontent.com/119438006/221022927-4bb9527b-9ab5-4823-8c84-4dc7185cdd9b.png)
There

**Comment by headshot2017** [comment]
here's an exe with -O0 flag (optimizations disabled):
[OpenLara_mingw.zip](https://github.com/headshot2017/OpenLara/files/10818298/OpenLara_mingw.zip)
if that fails try releases v1.01 and v1.0

**Comment by Ultracat3** [comment]
All failed, I'm sorry for taking so much time from you

**Comment by headshot2017** [comment]
it's ok! that's what github issues are for. meanwhile you can play around with the exe i sent you from the cdn discord link, however you'll also need to replace sm64.dll with the one from version 1.02 in case if it fails to launch

in the meantime i'll have to find a way to replicate your issue since i have a laptop with intel uhd graphics

**Comment by Ultracat3** [comment]
Ok, thanks
